straxen/contexts.py

```diff
@@ -207,3 +214,54 @@ def xenon1t_led(**kwargs):
         config=st.config,
         storage=st.storage,
         **st.context_config)
+
+nveto_common_opts = dict(
+    register_all=[
+       straxen.nveto_recorder,
+       straxen.nveto_pulse_processing,
+       # TPC plugins to avoid unused settings warning
+       straxen.pulse_processing,
+       straxen.peaklet_processing,
+       straxen.peak_processing,
+       straxen.event_processing
+    ],
+    store_run_fields=(
+        'name', 'number',
+        'reader.ini.name', 'tags.name',
+        'start', 'end', 'livetime',
+        'trigger.events_built'),
+    check_available=('raw_records_prenv',
+                     'raw_records_aqmonnv',
+                     'raw_records_nv',
+                     'lone_raw_records_nv',
+                     'lone_raw_record_statistics_nv',
+                     'records_nv',
+                     'pulse_edges_nv',
+                     'pulse_basics_nv'
+                     ))
+
+def strax_nveto_hdm_test(dynamic_voltage_05V=False, output_folder='./strax_data'):
+    if dynamic_voltage_05V:
+        config = {**xnt_common_config, **hdm_daqreader, **hdm_0_5V}
+    else:
+        config = {**xnt_common_config, **hdm_daqreader}
+    return strax.Context(
+        storage=[
+            strax.DataDirectory(
+                '/dali/lgrandi/wenz/strax_data/HdMdata_strax_v0_9_0/strax_data',
+                provide_run_metadata=False,
+                readonly=True),
+            strax.DataDirectory(output_folder,
+                                provide_run_metadata=False)],
+        config=config,
+        **nveto_common_opts)
+
+# HdM test specific configurations;
```

Comment:
Do these need to be defined as top-level globals? If they are only used inside the function above, you could specify them there.

Reply:
Done